README.md

```diff
@@ -6,41 +6,38 @@ maintains a set of components under the
 [kubernetes-csi](https://github.com/kubernetes-csi) GitHub
 organization.
 
-This repository contains code and documentation for building a
-kubernetes-csi release.
-
-
```

Comment:
With the decentralized approach, we might not need this "csi-release" at all anymore.